### ISSUE TITLE
Documentation elasticsearchDisk

### ISSUE THREAD

**Issue by a-nldisr** [mesos/elasticsearch#265]
Hi, 

Nothing serious, but want to notify you that.

The documentation states: 
```
    --elasticsearchDisk
       The amount of Disk resource to allocate to the elasticsearch instance
       (MB).
       Default: 1024.0
```

While the UI shows GB, when i configured the disk size in MB i got as result:
```
204800GB
DISK
```

**Comment by jrslv** [comment]
Hi and thanks for the report. The disk allocation in GUI is indeed displayed in gigabytes and there's already bugfix for the conversion as part of this pull request: 

https://github.com/mesos/elasticsearch/pull/256/files#diff-1a5c3fa30108a9f58cbcf62b1e5856a5R29

It is supposed to be merged before 0.3.0 is released.

Cheers

**Comment by a-nldisr** [comment]
Thanks!
